### ISSUE TITLE
The --check option should be default

### ISSUE THREAD

**Issue by abompard** [fedora-infra/fas2ipa#22]
Checking for conflicts should be on by default, especially if the conflicts file has been specified.

**Comment by nphilipp** [comment]
Hmm, if you don't specify any of `--pull`, `--push`, `--check`, it defaults to all three. What it doesn't do (and should) is always default to check even if `--pull` or `--push` is specified. However, I get the following error when running the script in the Vagrant box against a clean IPA:

````
(fas2ipa-vs8V2ZPt-py3.9) bash-5.0$ fas2ipa -u nphilipp
Logged into FAS (fedora): https://admin.fedoraproject.org/accounts
Logged into FAS (centos): https://accounts.centos.org
Logged into IPA
Pulling group information from FAS (fedora)...
Got 1261 groups!
Pulling group information from FAS (centos)...
Got 30 groups!
[fedora] finding user 'nphilipp'
[centos] finding user 'nphilipp'
Checking for conflicts between users from different FAS instances
Conflicts for user nphilipp:
	Conflicting email addresses between FAS instances:
		nphilipp@redhat.com: fedora
		nils@redhat.com: centos
Done checking user conflicts.
Found 1 users with conflicts.
Checking for conflicts between groups from different FAS instances
Conflicts for group accounts:
	Same group name between: fedora, centos
Conflicts for group accounts-moderators:
	Same group name between: fedora, centos
Done checking group conflicts.
Found 2 groups with conflicts.
Creating Agreements
Agreement with name "Fedora Project Contributor Agreement" already exists
Traceback (most recent call last):
  File "/home/vagrant/.cache/pypoetry/virtualenvs/fas2ipa-vs8V2ZPt-py3.9/bin/fas2ipa", line 33, in <module>
    sys.exit(load_entry_point('fas2ipa', 'console_scripts', 'fas2ipa')())
  File "/home/vagrant/.cache/pypoetry/virtualenvs/fas2ipa-vs8V2ZPt-py3.9/lib64/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/vagrant/.cache/pypoetry/virtualenvs/fas2ipa-vs8V2ZPt-py3.9/lib64/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/vagrant/.cache/pypoetry/virtualenvs/fas2ipa-vs8V2ZPt-py3.9/lib64/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/vagrant/.cache/pypoetry/virtualenvs/fas2ipa-vs8V2ZPt-py3.9/lib64/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/vagrant/fas2ipa/cli.py", line 222, in cli
    agreements_mgr.push_to_ipa()
  File "/vagrant/fas2ipa/agreements.py", line 55, in push_to_ipa
    self._create_agreement(agreement["name"], agreement_description, group_name)
  File "/vagrant/fas2ipa/agreements.py", line 37, in _create_agreement
    self.ipa.automember_add(group_name, o_type="group")
AttributeError: 'ClientLegacy' object has no attribute 'automember_add'
(fas2ipa-vs8V2ZPt-py3.9) bash-5.0$
````

@abompard did you see this error in your tests? Seems like we use a mix of legacy and current API on a legacy IPA client object. I plan to fix this along with making `--check` the default.

**Comment by abompard** [comment]
Fixed in 9fb12f7 that I forgot to push, sorry.
